README.md

```diff
@@ -2,37 +2,37 @@
 
 ## Overview
 
-This repo contains a starter project for a simple key/value database server. During the interview exercise, you will pair with another Software Engineer to extend the functionality of the server.
+This repository contains the beginnings of a key/value database server. During the interview, you will pair with another engineer to extend the server's functionality. This server is available in both Python and JavaScript variants.
 
-The starter project is available in both Python and Javascript variants.
+For more information on the web frameworks used, please refer to the [Flask](https://flask.palletsprojects.com/en/1.1.x/) and [Restify](http://restify.com/docs/home/) documentation. Also, many thanks to [The Recurse Center](https://www.recurse.com/pairing-tasks) for providing the inspiration for this exercise.
 
-For more information on the web frameworks used, please refer to the [Flask](https://flask.palletsprojects.com/en/1.1.x/) and [Restify](http://restify.com/docs/home/) documentation.
-
-Many thanks to [The Recurse Center](https://www.recurse.com/) for the inspiration for this exercise.
-
-## Getting Started
+## Getting started
 
 ### Dependencies
 
-The runtimes and libraries required to run either sample are contained in Docker images built from this repo. To run these examples, you must have the following tools installed:
+The runtimes and libraries required to run each server are encapsulated within container images. The instructions used to create those container images are contained within this repository. To run the servers, you must have the following dependencies installed:
 
-* Docker 19.03+
-* Docker Compose 1.17+
+- Docker 19.03+
+- Docker Compose 1.24+
```

Comment:
I bumped this because a version of Docker Compose newer than 1.17 comes bundled with up-to-date versions of the Docker engine.